GEOSagcm_GridComp/GEOSphysics_GridComp/GEOSsurface_GridComp/GEOSland_GridComp/GEOScatchCN_GridComp/Shared/catchmentCN.F90

```diff
@@ -2247,7 +2246,7 @@ SUBROUTINE WUPDAT (                                                   &
              ! same approach as for RZFLW (see subroutine RZDRAIN for
              ! comments)
              ZBAR1  = catch_calc_zbar( BF1(N), BF2(N), CATDEF(N) )  
-             SYSOIL = (2.*bf1(N)*amin1(amax1(zbar1,0.),PEATCLSM_ZBARMAX_4_SYSOIL) + 2.*bf1(N)*bf2(N))/1000.
+             SYSOIL = (2.*bf1(N)*amin1(amax1(zbar1,0.),0.45) + 2.*bf1(N)*bf2(N))/1000.
```

Comment:
This should become and "if statement" where you have the three porosity cases, I assume.